cli/client.js

```diff
@@ -40,7 +44,14 @@ if (!target) showError('missing target argument')
 const port = argv._[2]
 if (!port) showError('missing port argument')
 
-startClient(tunnel, target, port, (err) => {
+let headers = {}
+const header_items = Array.isArray(argv.header) ? argv.header : [argv.header]
+for (const item of header_items) {
+	const [key, ...data] = item.split(':');
```

Comment:
Let's follow [curl's method]() here and just split at the first `:\s*`.

```suggestion
	const match = /(?<=\w):\s+(?=\w)/.exec(item);
	if (!match) showError(`invalid --header value: '${item}'`)
	const name = item.slice(0, match.index)
	const val = item.slice(match.index + match[0].length)
	headers[name] = val
```

Reply:
i test and i dont think it'is generic enough:
1. use `\s*` rather than `\s+` in your demo
2. mismatch case: `Cookie: .anything` (data start with `.`)
why not using `split`?

Reply:
using a regex is not a must. i would just like to follow curl here.